tests/dragonfly/generic_test.py

```diff
@@ -168,3 +169,41 @@ async def test_denyoom_commands(df_factory):
 
     # mget should not be rejected
     await client.execute_command("mget x")
+
+
+@pytest.mark.parametrize("type", ["list", "hash", "string", "set", "zset"])
```

Comment:
we can do this test in unit test
see TEST_F(RdbTest, LoadHugeSet) for example
btw we can check the dump and restore command directly

Reply:
It's cleaner and with the seeder (your suggestion) also easier to do in pytests :)